ext/dapr-ext-workflow/setup.cfg

```diff
@@ -25,6 +25,7 @@ packages = find_namespace:
 include_package_data = True
 install_requires =
     dapr-dev >= 1.9.0rc1.dev
+    durabletask >= 0.1.0a1
```

Comment:
That version does not seem to exist which is why all your tests are failing.

Reply:
Now, the version of durabletask on PyPi is supporting 3.7+.
build is passing for all versions now.